napari/layers/_image_layer/model.py

```diff
@@ -312,17 +298,16 @@ def method(self, method):
     def interpolation(self):
         """string: Equipped interpolation method's name.
```

Comment:
this docstring should include all possible methods listed in set format as the type: `{'nearest', 'bicubic', ...}`

Reply:
Good point. Fixed!